enclave-runtime/src/test/evm_pallet_tests.rs

```diff
@@ -0,0 +1,238 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+use crate::test::tests_main::test_setup;
```

Comment:
This creates a circular dependency. We should put the `test_setup` function in a separate module, in the `fixtures` folder.

Reply:
Shouldn't this be imported from fixtures now?

Reply:
of course!~~!